examples/tinywl/CMakeLists.txt

```diff
@@ -16,6 +16,8 @@ if(PROJECT_IS_TOP_LEVEL) # if build as a separated project
     else()
         find_package(QT NAMES Qt6 REQUIRED COMPONENTS Core)
     endif()
+else()
+    ws_generate(server wayland-protocols stable/xdg-shell/xdg-shell.xml xdg-shell-protocol)
```

Comment:
why needs generate in examples?

Reply:
CMAKE_CURRENT_BINARY_DIR are different for qwlroots and tinywl